src/go/k8s/config/crd/bases/cluster.redpanda.com_redpandas.yaml

```diff
@@ -1,9 +1,11 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.13.0
+    controller-gen.kubebuilder.io/version: v0.4.1
```

Comment:
Oh no, this needs to be reverted

Reply:
sigh....
we also need to updated our tooling internally.